org.dharkael.Flameshot.yaml

```diff
@@ -0,0 +1,36 @@
+app-id: org.dharkael.Flameshot
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: flameshot
+rename-icon: flameshot
+rename-appdata-file: flameshot.appdata.xml
+rename-desktop-file: flameshot.desktop
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=x11
```

Comment:
I'm not exactly sure but `fallback-x11` probably implies `--socket=x11`.

Reply:
removed `fallback-x11`